### ISSUE TITLE
--views-directory option doesn't allow set custom directory structure as intended.

### ISSUE THREAD

**Issue by unknownbreakdown** [CrestApps/laravel-code-generator#159]
_--views-directory_ option doesn't allow set custom directory structure under _resources/views_.  It always create first as parent directory the plural name of the model.

eg. _--views-directory=management/categories_ generates _categories/management/categories/_ directory structure under resources/views/

**Comment by unknownbreakdown** [comment]
A workaround is to use the _--routes-prefix_ option instead of the _--views-directory_ one

**Comment by AjithLalps** [comment]
Same issue with me.

**Comment by mgip** [comment]
Yes, me too... i try to use --views-directory but, this arg work only inside the resource folde inside resources/views...
I need change resources/views into resources/views/child for all generation.
